src/content/docs/en/reference/errors/redirect-with-no-location.mdx

```diff
@@ -17,6 +17,6 @@ import DontEditWarning from '~/components/DontEditWarning.astro'
 A redirect must be given a location with the `Location` header.
 
 **See Also:**
--  [Astro.redirect](/en/reference/api-reference/#redirect)
+-  [Astro.redirect](/en/reference/api-reference/#astroredirect)
```

Comment:
So close!

Reply:
Ack! lol I did a search for #asto and this one never came up! 😅